js/tweek-local-cache/spec/behavior/tweek-repository.spec.ts

```diff
@@ -1,15 +1,43 @@
 import { expect } from 'chai';
 import getenv = require('getenv');
 import { createTweekClient, ITweekClient, Context } from 'tweek-client';
+import axios from 'axios';
 import MemoryStore from '../../src/memory-store';
 import TweekRepository from '../../src/tweek-repository';
+import { delay } from '../../src/utils';
 
 const TWEEK_LOCAL_API = getenv.string('TWEEK_LOCAL_API', 'http://127.0.0.1:1111');
 
-describe('tweek repo behavior test', () => {
+describe('tweek repo behavior test', function() {
+  this.timeout(180000);
+
   let _tweekRepo: TweekRepository;
   let _tweekClient: ITweekClient;
 
+  before(async () => {
+    const instance = axios.create({ baseURL: TWEEK_LOCAL_API, timeout: 2000 });
+
+    let error;
+    for (let i = 0; i < 20; i++) {
+      try {
+        await instance.get('/health');
+        const result = await instance.get('/api/v1/keys/@tweek_clients_tests/_');
+        expect(result.data).to.deep.include({
```

Comment:
we're checking for `include` not `equal`, so it will only break if these specific keys were changed, but if they change the tests themselves will break too, so you will still need to fix the tests